extensions/mssql/package.json

```diff
@@ -165,6 +165,7 @@
         "axios": "^1.12.0",
         "dotenv": "^16.4.5",
         "error-ex": "^1.3.0",
+        "excel-builder-vanilla": "^4.2.1",
```

Comment:
Given that we're already able to save to excel files elsewhere in the extension, I assume we have existing code that doesn't require a new package?

Reply:
`excel-builder-vanilla` is a package that I also maintain, a fork of `excel-builder`, and that can be used in Slickgrid-React when enabling grid Excel export (you would also need `@slickgrid-universal/excel-export`). So that might be the reason to choose it!? `excel-builder-vanilla` is also the smallest JS library available, it's ESM and 7x smaller than other libs like XLXS